cmd/database.go

```diff
@@ -634,7 +634,7 @@ func prepPostgreSQL(d *DDConfig, t *targetOS) error {
 	_, err = runPgSQLCmd(d, createUsr)
 	if err != nil {
 		d.traceMsg("Failed to create database user for DefectDojo")
-		return err
+		d.traceMsg("Continuing after error creating user, non-fatal error")
```

Comment:
I know nothing about Defect Dojo, but is there any chance we can PR this change upstream rather than maintaining a fork?

Reply:
Based on documentation for the godojo repo I don't think this aligns with how they want it to operate, but I can also try. For now though using this fork is our best option.

Reply:
Do we know why the `CREATE USER` command is failing? It seems like the given query should succeed in RDS

Reply:
it seems like we should either handle this in the sql or interrogate the error message to be sure that we're only ignoring this category of failure

Reply:
This covers an issue where if the user already exists the code errors out and won't continue the setup process. It basically does the same thing as earlier in the code where they allow the code to continue even if they are unable to delete a user.